packages/components/src/Link/Link.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactNode, forwardRef } from "react";
+import Clickable, { ClickableProps } from "../Clickable";
+
+type LinkHTMLElementProps = Omit<
+  React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  "disabled"
+>;
+
+export type LinkProps = LinkHTMLElementProps & {
+  as?: "a" | React.ComponentType<any>;
+  /**
+   * The link contents or label.
+   */
+  children: ReactNode;
+  color?: ClickableProps<"button">["color"];
```

Comment:
should this be `"a"` instead of `"button"`?

Reply:
Good call, thanks!